spec/routing/announcements_routing_spec.rb

```diff
@@ -1,14 +1,12 @@
 require 'rails_helper'
 
 describe 'routes for announcements', type: :routing do
-  routes { Basechurch::Engine.routes }
-
   describe 'POST /v1/announcements' do
     let(:id) { 1 }
     let(:post_announcement) { post "/v1/announcements" }
 
     it 'routes to announcements#create)' do
-      expect(post_announcement).to route_to(controller: 'basechurch/v1/announcements',
+      expect(post_announcement).to route_to(controller: 'v1/announcements',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.